pombola/south_africa/views/attendance.py

```diff
@@ -81,21 +81,24 @@ def filter_attendance(self, annual_attendance, party, position):
             pa_url = 'http://www.pa.org.za'
             for position in active_minister_positions:
                 minister = position.person
-                if minister.slug in active_minister_slugs:
-
-                    # Build name consistent with PMG data
-                    initials = "".join(name[0].upper() for name in minister.given_name.split())
-                    minister_name = "{}, {} {}".format(
-                        minister.family_name, minister.title, initials)
-
-                    minister_attendance.append({
-                        'member': {
-                            'name': minister_name,
-                            'pa_url': pa_url + minister.get_absolute_url(),
-                            'party_name': minister.parties()[0].slug.upper()},
-                        'attendance': {
-                            'P': 0}
-                    })
+                if party and minister.parties()[0].slug.upper() != party:
```

Comment:
Sorry, I don't understand this - could you add a comment in the code explaining why the minister is skipped in this case?